Mathlib/NumberTheory/ZetaFunction.lean

```diff
@@ -797,3 +797,39 @@ theorem riemannZeta_neg_nat_eq_bernoulli (k : ℕ) :
     rw [inv_mul_cancel (ofReal_ne_zero.mpr <| pow_ne_zero _ pi_pos.ne'),
       inv_mul_cancel (ofReal_ne_zero.mpr <| pow_ne_zero _ two_ne_zero), one_mul, one_mul]
 #align riemann_zeta_neg_nat_eq_bernoulli riemannZeta_neg_nat_eq_bernoulli
+
+/-- The residue of `Λ(s)` at `s = 1` is equal to 1. -/
```

Comment:
You may need to `Open Topology` for the neighborhood notation.

Reply:
I don't think so – there's `open scoped Topology` at the head of the file. It compiles ok on my machine, so there shouldn't be any imports missing.

Reply:
OK. I copied the new stuff into a test file opening what `#where` in the original file said, but apparently `#where` does not show `Open scoped ...` stuff.